Gopkg.lock

```diff
@@ -658,6 +660,16 @@
   ]
   revision = "01bc873149a1802eb74df583613872d126449ed5"
 
+[[projects]]
+  branch = "master"
+  name = "k8s.io/apiserver"
```

Comment:
Why do we need this?

Reply:
We dont; I just missed to run `dep ensure` at the end ;)